File: 프로그래머스/lv0/120849. 모음 제거/모음 제거.cpp
#include <string>
#include <algorithm>
#include <vector>

using namespace std;

string solution(string my_string)
{
    char vowel[5] = {'a', 'e', 'i', 'o', 'u'};
    
    for (int i = 0; i < 5; i++)
        my_string.erase(remove(my_string.begin(), my_string.end(), vowel[i]), my_string.end());
    
    return my_string;
}

File: 프로그래머스/0/120849. 모음 제거/모음 제거.java
class Solution {
    
    static final String[] vowels = {"a", "e", "i", "o", "u"};
    
    public String solution(String my_string) {
        for (String vowel : vowels) {
            my_string = my_string.replaceAll(vowel, "");
        }
        return my_string;
    }
}